app/views/reviews/new.html.erb

```diff
@@ -84,7 +84,12 @@
 </div>
 
 <% if @submission.review_status == 'pass' %>
+  <div id='move-submission-form-box'>
+  </div>
   <div class="alert alert-info">
-    <h1>This submission has already been marked as passing.</h1>
+    <h1>
+      This submission has already been marked as passing.
+      <%= link_to 'move submission', submission_path(@submission), remote: true, class: 'btn btn-xs btn-default', id: 'show-move-submission-form-button' %>
```

Comment:
Why use AJAX here? Why not just load the form and hide it with CSS, then show it with JavaScript?

Reply:
I originally had it just hiding the form, but then it's loading a bunch of extra stuff from the db every time someone wants to pull up the review page, when it'll only be used a fraction of the time. So I figured using AJAX or an entirely new route would be preferable?

Reply:
Is it an expensive DB operation? If so, that makes sense.`edit` would be a more appropriate controller action.